Doberman/AlarmMonitor.py

```diff
@@ -45,7 +44,7 @@ def send_email(self, toaddr, subject, message, cc=None, bcc=None, add_signature=
             return -1
         try:
             # Compose connection details and addresses
-            now = datetime.datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S')
+            now = dtnow().replace(tzinfo=timezone.utc).astimezone(tz=None).strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
Rather than all the calls we make to `strftime` we should use `isoformat`, also in L162 and L225